CRM/Report/Form/Contact/Summary.php

```diff
@@ -147,17 +147,11 @@ public function from() {
   }
 
   public function postProcess() {
-
     $this->beginPostProcess();
-
-    // get the acl clauses built before we assemble the query
-    $this->buildACLClause($this->_aliases['civicrm_contact']);
-
+    $this->buildPermissionClause();
```

Comment:
It looks like this gets called already from buildQuery() right below? If you just remove the line does that fix the problem?

Reply:
You are right. It works just removing the buildACLClause line